### ISSUE TITLE
Assert when trying to compile a project

### ISSUE THREAD

**Issue by thradams** [PascalBeyer/Headerless-C-Compiler#1]
I am trying to compile this source

https://godbolt.org/z/r5hf4xYKo

When I compile it gives me

C:\Projects\Headerless-C-Compiler\src\coff_writer.c(2460): ASSERT FIRED: 'decl->relative_virtual_address > 0' in function print_coff.

Any idea?

**Comment by PascalBeyer** [comment]
I have checked it out and the issue is in line 37085:
```c
WINBASEAPI unsigned long WINAPI GetEnvironmentVariableA(const char* name,
char* buffer,
unsigned long size);
```
Currently, the compiler does not support support `__declspec(dllimport)` declarations  inside of a function.
I will push a hotfix for this and for debugging not working with Visual Studio later today.

**Comment by thradams** [comment]
Yes...I moved the declaration and it worked! amazing; congratulations.

**Comment by PascalBeyer** [comment]
"Fixed" with `v0.2.1`.
